### DISCUSSION TITLE
INSTALL_PACKAGES and PACKAGES_MIRROR

### DISCUSSION THREAD

**Discussion by RunyonR** [jlesage/docker-putty#10] [General]
Logs show that the container keeps failing trying to install packages.  I'm not 100% sure what I need to put in these to env variables.  I currently have both set to "https://mirrors.itwrx.org/alpine"

Below is the log output.

![image](https://github.com/jlesage/docker-putty/assets/122032098/d5c693a1-3573-4658-a027-89ee8f10984f)

If someone could share what they have for these variables, I can probably figure it out from there.

**Comment by jlesage** [comment]
What are you trying to achieve exactly ?  These variables don't have to be set if not required.

**Reply by RunyonR** [reply]
I'm setting this up on a Synology NAS and it tells me that they are required.  I suppose I may be able to delete them completely.

**Reply by jlesage** [reply]
Correct, you can remove them.
See https://github.com/jlesage/docker-putty?tab=readme-ov-file#deployment-considerations
